repo/packages/J/jenkins/18/resource.json

```diff
@@ -0,0 +1,14 @@
+{
+  "images": {
+    "icon-small": "https://downloads.mesosphere.com/universe/assets/icon-service-jenkins-small.png",
+    "icon-medium": "https://downloads.mesosphere.com/universe/assets/icon-service-jenkins-medium.png",
+    "icon-large": "https://downloads.mesosphere.com/universe/assets/icon-service-jenkins-large.png"
+  },
+  "assets": {
+    "container": {
+      "docker": {
+        "jenkins-303-2323": "mesosphere/jenkins:3.1.0-2.46.2"
```

Comment:
Should change this to `jenkins-310-2462` and update its reference in `marathon.json.mustache`.